services/madoc-ts/src/frontend/site/features/admin/CanvasTaskProgress.tsx

```diff
@@ -84,6 +84,7 @@ export const CanvasTaskProgress: React.FC = () => {
 
   const [deleteCaptureModel, deleteCaptureModelStatus] = useMutation(async () => {
     if (canvasModel && canvasModel.model && canvasModel.model.id) {
+      await api.batchDeleteTasks({ resourceId: canvasId, subject: canvasTask?.subject });
```

Comment:
This might delete tasks outside of the current project at the same time. The project should contain a `root_task_id` that can be used (can't remember the exact API)